dataland-frontend/src/components/forms/parts/elements/derived/ProcurementCategoryFormElement.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+  <div class="form-field">
+    <div data-test="dataPointToggle" class="form-field border-none vertical-middle">
+      <InputSwitch
+        data-test="dataPointToggleButton"
+        inputId="dataPointIsAvailableSwitch"
+        @click="this.isItActive = false"
+        v-model="isItActive"
```

Comment:
I have a feeling that a data-binding is sufficient here.
Because you have to cases:

Case A) You click on the switch to set `isItActive = true`.   This is already covered by the click event.

Case B) `isItActive` is set to `true` by loading data.  This should be covered by a data binding here.

=> Why is this a v-model?  Is this necessary?

Reply:
Alternatively you could remove the click event and just leave the v-model?   I am not sure.

Reply:
Done.

Reply:
Has someone validated, that it still works after this change?

Reply:
yes, I checked. it works